### DISCUSSION TITLE
How to do merge column (colSpan) in table?

### DISCUSSION THREAD

**Discussion by sittiponghaus** [remarkjs/remark#656] [Q&A] [answered]
How to do merge column `colSpan` in the table?

Or this isn't supported yet?

Thank you in advance.

Edited: conSpan -> colSpan

**Comment by wooorm** [comment, marked as answer]
I do not know what `conSpan` is.

Please spend some time framing your question. See here for help: https://github.com/remarkjs/.github/blob/main/contributing.md#support

**Reply by sittiponghaus** [reply]
I apologize for my keyboard autocompleted. I mean `colSpan` which merged multiple columns in single column like this.

![image](https://user-images.githubusercontent.com/2971450/110639797-b5587f80-81e2-11eb-9830-9955ef832d95.png)

**Reply by wooorm** [reply]
Thanks for the update. Unfortunately, setting a `colSpan` is not supported in markdown.

**Reply by sittiponghaus** [reply]
Thank you for your support.
